docs/09-bootstrapping-kubernetes-workers.md

```diff
@@ -88,8 +88,8 @@ Install the worker binaries:
 ### Configure CNI Networking
 
 Create the `bridge` network configuration file replacing POD_CIDR with address retrieved initially from Azure VM tags:
-(Note: The Azure Metadata Instance Service is used to retrieve the POD_CIDR tag for each worker.
-https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/virtual-machines/windows/instance-metadata-service.md)
+
+> Note: the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service) is used to retrieve the POD_CIDR tag for each worker.
```

Comment:
I thought it might be more consistent if this doc link referenced the actual docs.microsoft.com location instead of GitHub